.devcontainer/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04
+
+# export PATH=/usr/local/cuda-12.1/bin/:$PATH
+# cd src/c
+# mkdir build
+# cd build
+# cmake -DCMAKE_CXX_COMPILER=clang++ -DLLVM_DIR=/usr/lib/llvm-10/cmake ..
```

Comment:
@vsoch Do we need these lines that are commented out or can they be removed?

Reply:
I left those there to show how to do the build, which is sometimes helpful if you move the Dockerfile outside of the developer environment. I would say it doesn't hurt to leave them so the Dockerfile can speak for itself alongside the code.

Reply:
Sounds good, thank you!